lua/lspconfig/server_configurations/sumneko_lua.lua

```diff
@@ -16,17 +16,14 @@ https://github.com/sumneko/lua-language-server
 
 Lua language server.
 
-`lua-language-server` can be installed by following the instructions [here](https://github.com/sumneko/lua-language-server/wiki/Build-and-Run).
-
-**By default, lua-language-server doesn't have a `cmd` set.** This is because nvim-lspconfig does not make assumptions about your path. You must add the following to your init.vim or init.lua to set `cmd` to the absolute path ($HOME and ~ are not expanded) of your unzipped and compiled lua-language-server.
+`lua-language-server` can be installed by following the instructions [here](https://github.com/sumneko/lua-language-server/wiki/Build-and-Run). The default `cmd` assumes that the `lua-language-server` binary can be found in `$PATH`.
```

Comment:
maybe we should mention the binary is found under `lua-language-server/...(I forget)PLATFORM/lua-language-server`

Reply:
It's already explicitly mentioned in the last part of the linked build instructions? (Avoid adding new source of truth, and all that...)

Reply:
Yes, I know, but I think recent evidence says it's not obvious.

Reply:
I hear you; but I suspect that people following the upstream build instructions to obtain their language server are _not_ the people having problems finding the binary afterwards...

Reply:
It's fine, if people complain we will add it :)